bedrock/base/templates/base-pebbles.html

```diff
@@ -127,6 +127,14 @@
     <!--[if !lte IE 8]><!-->
     {% block js %}{% endblock %}
 
+    {% if self.structured_data()|trim|length or self.structured_data()|trim|length %}
```

Comment:
Hmm, what's the need for repeating `self.structured_data()|trim|length` using an `or`?

Also, should this be repeated in each base template for consistency?

Reply:
Oh, heh, that's probably a copy pasta error.

`base-article`, `base-protocol`, and `base-protocol-mozilla` all inherit from this template. But `base-resp` does not and I made a bad assumption that it did, so thank you.